pkg/cloudprovider/providers/vsphere/vclib/volumeoptions.go

```diff
@@ -33,6 +33,7 @@ type VolumeOptions struct {
 	StoragePolicyName      string
 	StoragePolicyID        string
 	SCSIControllerType     string
+	Zone                   []string
```

Comment:
What is the purpose of this property?

Reply:
This property captures the zones specified in allowedTopology of SC. This property is read from volumeOptions in vsphere.go CreateVolume(volumeOptions *vclib.VolumeOptions) and processed for datastore placement.